packages/odata-v2/src/request-builder/response-data-accessor.ts

```diff
@@ -1,5 +1,6 @@
 import { createLogger } from '@sap-cloud-sdk/util';
-import { ResponseDataAccessor } from '@sap-cloud-sdk/odata-common';
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
Could you try to use something like: (https://eslint.org/docs/user-guide/configuring/configuration-files)

    'import/no-internal-modules': ["error", {
      "allow": ["@sap-cloud-sdk/**/*"],
    }]

instead of adding very often the ignore case

Reply:
I agree, I didn't see such a rule in the other PRs and assumed this was the go-to approach. Good to know this can be made cleaner.